src/main/resources/migrations/common/V2024.04.22__baseline_tables.sql

```diff
@@ -0,0 +1,328 @@
+-- The probation teams involved in video link bookings.
+-- A probation team has contacts who will be notified of bookings, and changes to them.
+-- A probation team has users who have selected to view/add bookings for it.
+-- A probation team has video bookings related to it.
+----------------------------------------------------------------------------------------
+
+CREATE TABLE probation_team
+(
+    probation_team_id    bigserial NOT NULL CONSTRAINT probation_team_id_pk PRIMARY KEY,
+    code                 varchar(40) NOT NULL UNIQUE,
+    description          varchar(100) NOT NULL,
+    enabled              boolean NOT NULL,
+    notes                varchar(100) NOT NULL,
+    created_by           varchar(100) NOT NULL,
+    created_time         timestamp NOT NULL,
+    amended_by           varchar(100),
+    amended_time         timestamp
+);
+
+CREATE INDEX idx_probation_team_code ON probation_team(code);
+
+---------------------------------------------------------------------------------------
+-- The people involved in video link bookings for a probation team.
+-- This can be a single group email contact or a list of individual contacts.
+-- This list of contacts will be notified of bookings for this team, and any changes.
+----------------------------------------------------------------------------------------
+
+CREATE TABLE probation_team_contact
+(
+    probation_team_contact_id bigserial NOT NULL CONSTRAINT probation_team_contact_id_pk PRIMARY KEY,
+    probation_team_id    bigint NOT NULL REFERENCES probation_team(probation_team_id),
+    name                 varchar(100) NOT NULL,
+    email                varchar(100),
+    telephone            varchar(20),
+    position             varchar(100),
+    enabled              boolean NOT NULL,
+    notes                varchar(200),
+    created_by           varchar(100) NOT NULL,
+    created_time         timestamp NOT NULL,
+    amended_by           varchar(100),
+    amended_time         timestamp
+);
+
+CREATE INDEX idx_probation_contact_team_id ON probation_team_contact(probation_team_id);
+CREATE INDEX idx_probation_contact_name ON probation_team_contact(name);
+CREATE INDEX idx_probation_contact_email ON probation_team_contact(email);
+
+---------------------------------------------------------------------------------------
+-- Reference table for prisons - populate from register
+---------------------------------------------------------------------------------------
+
+CREATE TABLE prison
+(
+    prison_id    bigserial   NOT NULL CONSTRAINT prison_id_pk PRIMARY KEY,
+    code         varchar(10) NOT NULL UNIQUE,
+    name         varchar(60) NOT NULL,
+    description  varchar(100) NOT NULL,
+    enabled      boolean NOT NULL,
+    notes        varchar(200),
+    created_by   varchar(100) NOT NULL,
+    created_time timestamp NOT NULL,
+    amended_by   varchar(100),
+    amended_time timestamp
+);
+
+CREATE INDEX idx_prison_code ON prison(code);
```

Comment:
Add the UNIQUE here instead of on the `code` column? 🤷

Reply:
Yes. Will add now, thanks